src/content/en/updates/2017/07/upcoming-regexp-features.md

```diff
@@ -0,0 +1,213 @@
+project_path: /web/_project.yaml
+book_path: /web/updates/_book.yaml
+description: This update discusses upcoming features in JavaScript RegExps.
+
+{# wf_updated_on: 2017-07-10 #}
+{# wf_published_on: 2017-07-10 #}
+{# wf_tags: javascript #}
+
+{# wf_featured_image: /web/updates/images/2017/07/v8.png #}
+{# wf_featured_snippet: An overview of the exciting new features coming to
+JavaScript RegExps, including named captures, the dotAll flag, Unicode property
+escapes, and lookbehind assertions. #}
+
+{% include "web/tools/chrome-devtools/_shared/styles.html" %}
+
+
+# Upcoming RegExp Features {: .page-title }
+
+{% include "web/_shared/contributors/jgruber.html" %}
+{% include "web/_shared/contributors/mathiasbynens.html" %}
+{% include "web/_shared/contributors/yangguo.html" %}
+
+Regular expressions, or RegExps, are an important part of the JavaScript
+language. When used properly, they can greatly simplify string processing.
+
+ES2015 introduced many new features to the JavaScript language, including
+significant improvements to the regular expression syntax with the Unicode (`/u`)
+and sticky (`/y`) flags. But development has not stopped since then — in tight
+collaboration with other members at TC39 (the ECMAScript standards body), the
+V8 team has proposed and co-designed several new features to make RegExps even
+more powerful.
+
+These features are currently being proposed for inclusion in the JavaScript
+specification. Even though the proposals have not been fully accepted, they are
+already at [Stage 3 in the TC39
+process](https://tc39.github.io/process-document/). We have implemented these
+features behind flags (see below) in order to be able to provide timely design
```

Comment:
I went with the 'behind a flag' suggestion, as mentioning a '--harmony' flag might be confusing for readers. The next paragraph clarifies how to enable the flag.